index.js

```diff
@@ -1,51 +1,111 @@
 'use strict';
 const alfy = require('alfy');
+const utils = require('./js/utilities');
+const formatting = require('./js/formatting');
+const baseUrl = 'https://api.darksky.net/forecast/';
+const excludes = '?exclude=minutely,hourly&units=auto';
 
-//fetches the weather for Valencia
-alfy.fetch('https://api.darksky.net/forecast/b3bfdcda400630b909f112b1530aea93/37.8267,-122.4233', {
-    query: {
-        q: alfy.input
+//fetches the weather, based on input or default settings
+try {
+    
+    var mapsAPIKey = alfy.cache.get('Google Maps API Token');
```

Comment:
- You can use `let` or `const`
- You should add and configure a linter to help you with issues like this.